update/sync/base.go

```diff
@@ -0,0 +1,29 @@
+package sync
```

Comment:
We have `base` here, `common` in the upper level package. We should stick with a convention (even if it's not the best approach, but I see we already use this pattern in other packages)

Reply:
common is used to keep all the keys and object type calculations in one place. will refactor.